### ISSUE TITLE
9fans.net has disappeared into thin air

### ISSUE THREAD

**Issue by diddledan** [9fans/go#5]
As per issue #3, I have been trying to install godef and running up against the problem indicated in their issue (https://github.com/rogpeppe/godef/issues/4) in addition to the issue above. The last comment in #3 says that the server is back in The Cloud but that it might disappear into thin air. So I am raising this issue to highlight that it has indeed disappeared into thin air. Might it be prudent to point the `9fans.net` package name to `github.com/9fans` instead?

TL/DR, suggested fix: change `9fans.net/go/*` to `github.com/9fans/go/*`

**Comment by rsc** [comment]
It's back. The comment about vanishing into thin air was just a bad joke about The Cloud, but it does seem to have come true. I will set up more monitoring. Thanks for your patience.

**Comment by nsajko** [comment]
It doesn't seem to be back ...?

**Comment by rsc** [comment]
https://9fans.net/go is working fine. What URL are you trying?

**Comment by nsajko** [comment]
oh, sorry; I took your comment to mean the whole site should be back up but there were a lot of pages on 9fans.net and swtch.com that return error 404.
